rook/migration/v1.11.x-to-v1.14.x.md

```diff
@@ -0,0 +1,85 @@
+# Upgrade rook-ceph to v1.11.x to v1.14.x
+
+## Prerequisites
+
+- [ ] Check the state of rook-ceph so it is healthy before you begin:
+
+  ```bash
+  helm -n rook-ceph list --all
+  # should show all deployed
+  kubectl -n rook-ceph get jobs
+  # should show all completed
+  kubectl -n rook-ceph get pods
+  # should show all running or completed
+  kubectl -n rook-ceph get cephclusters
+  # should show PHASE = Ready and HEALTH = HEALTH_OK
+  ```
+
+## Apply upgrade 1.12.x - _disruptive_
+
+- 1. Fetch chart version:
+
+```console
+sudo rm -rf helmfile.d/upstream/rook-ceph
```

Comment:
Should not need the sodu for these `rm`
```suggestion
rm -rf helmfile.d/upstream/rook-ceph
```

Reply:
> Looks like v1.11.x-to.v1.14.x have all the steps now. So this and the other readmes could be removed? 🙂

Yeah, will ask @OlleLarsson what he thinks will work!

Reply:
sudo should't be required.